src/main/scala/Collection.scala

```diff
@@ -0,0 +1,36 @@
+class Collection {
+  def ExSet(){
+    val set = Set(1,2,3,4)
+    val subSet = Set(5,7)
+
+    println(subSet ++ set)
+  }
+  def PatternMatch(food:String){
```

Comment:
method names start with a lowercase word, only words after should have capitalised first letter